packages/veritone-react-common/src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { shape, string, number, arrayOf, oneOfType, node, func } from 'prop-types';
+import { MenuList, Popover } from '@material-ui/core';
+import { ChevronRight, MoreVert } from '@material-ui/icons';
+import classNames from 'classnames';
+
+import BreadcrumbItem from './BreadcrumbItem';
+
+import styles from './Breadcrumbs.scss';
+
+export default class Breadcrumbs extends React.Component {
+  static propTypes = {
+    pathList: arrayOf(shape({
+      id: string.isRequired,
+    })),
+    maxItems: number,
+    seperator: oneOfType([string, node]),
+    onCrumbClick: func.isRequired
+  }
+
+  static defaultProps = {
+    maxItems: 5,
+    seperator: <ChevronRight className={styles['icon-color']} />,
+  }
+
+  state = {
+    openSpreadPath: false
+  }
+
+  onSpreadClick = (event) => {
+    this.anchorEl = event.target;
+    this.setState(({ openSpreadPath }) => ({
+      openSpreadPath: !openSpreadPath
+    }));
+  }
+
+  clickAway = () => {
+    this.anchorEl = null;
+    this.setState({
+      openSpreadPath: false
+    })
+  }
+
+  onCrumbClick = (event) => {
+    let elementData = Object.assign({}, event.target.dataset);
+    if (Object.keys(elementData).length === 0) {
```

Comment:
After you've used the closure func, you can just check to see if the crumb is defined, and pass the crumb directly to `this.props.onCrumbClick`